docs/getting-started/enterprise/key-connector.mdx

```diff
@@ -139,6 +178,12 @@ Run the following command from the repository root:
 dotnet run --project src/KeyConnector --configuration Development
 ```
 
+:::note
+
+If running on ARM based Mac you may need to use `/usr/local/share/dotnet/x64/dotnet`
```

Comment:
Would it be beneficial to provide a copyable full command?
`arch -x86_64 /usr/local/share/dotnet/x64/dotnet run --project src/KeyConnector --configuration Development`

Reply:
Just updated it but without the `arch -x86_64`, it's not needed when providing the full path, it could be used as `arch -x86_64 dotnet` but there were a few times that it didn't work for me so I tend to use the fullpath.